src/virtualenv/create/creator.py

```diff
@@ -90,6 +92,13 @@ def add_parser_arguments(cls, parser, interpreter, meta, app_data):
             help="remove the destination directory if exist before starting (will overwrite files otherwise)",
             default=False,
         )
+        parser.add_argument(
+            "--no-vcs-ignore",
+            dest="vcs_ignore",
+            action="store_false",
+            help="don't create VCS ignore directive in the destination directory",
+            default=True,
```

Comment:
Does this break convention and more importantly the CLI option-environment variable interop, by renaming the argument?  I wasn't aware of the config-file/env-var pathway to set these, and now it's not clear to me if those will expect a value at `no_vcs_ignore`/`VIRTUALENV_NO_VCS_IGNORE` or `vcs_ignore`/`VIRTUALENV_VCS_IGNORE` or some combo.

Reply:
So if you put:

```
[virtualenv]
no_vcs_ignore = false
```

in your users ``virtualenv.ini`` file you enabled this flag always on your machine.

Reply:
Perfect